doc/developers/maintainer.rst

```diff
@@ -211,6 +211,12 @@ Making a release
 
    https://github.com/scikit-learn/scikit-learn/actions?query=workflow%3A%22Publish+to+Pypi%22
 
+4.1 You can test the conda-forge builds by submitting a PR to the feedstock
+    repo: https://github.com/conda-forge/scikit-learn-feedstock. If you want to
+    publish an RC release on conda-forge, the PR should target the `rc` branch
```

Comment:
Could we make it in bold regarding the `rc` branch. I think this is something that could be easily overlooked.

Reply:
I personally don't check a rendered version of this doc, so I'm not sure making it bold would help.